Governance/Votes

```diff
@@ -0,0 +1,3 @@
+- There are two kinds of votes. Question votes and selection votes.
+- The possible answers for Question votes are ("abstain", "yes", "no")
+- In selection votes, voters have positive and negative voting power points.
```

Comment:
Do the types of votes relate in any way to the type of proposals? Do we have two types of proposals as well?